app/controllers/games_controller.rb

```diff
@@ -45,4 +56,26 @@ def load_game_graph_data
       { name: 'Challenges Solved', data: SolvedChallenge.group_by_hour(:created_at).count }
     ]
   end
+
+  def load_game_with_users
+    @game = Game.includes(:users).instance
+  end
+
+  private
+
+  # Creates a zip from any collection of files available on the user model.
+  # For example, create_zip_of('resume') will create a zip of all resumes
+  # uploaded by all teams, broken out into the team folders.
+  def create_zip_of(uploader)
+    compressed_filestream = Zip::OutputStream.write_buffer do |zos|
+      @game.users.each do |user|
+        next unless (file_contents = user.send(uploader).read)
+        zos.put_next_entry "#{uploader.pluralize}/#{user.team_id}/#{user.full_name}.pdf"
```

Comment:
Would it be better to use team name here instead of id?

Reply:
The team name is not sanitized and we don't have a way to guarantee that two team names don't simplify down to the same thing.